packages/template-retail-react-app/package.json

```diff
@@ -98,7 +99,7 @@
     },
     {
       "path": "build/vendor.js",
-      "maxSize": "325 kB"
+      "maxSize": "350 kB"
```

Comment:
"@salesforce/cc-datacloud-typescript" is only 2.1kb minified.. did we really need to bump this up that much?